incremental-font-transfer/Cargo.toml

```diff
@@ -15,16 +15,18 @@ repository.workspace = true
 all-features = true
 
 [features]
-default = ["read-fonts/std"]
+default = ["read-fonts/std", "c-brotli"]
 cli = ["clap", "regex"]
+rust-brotli = ["shared-brotli-patch-decoder/rust-brotli"]
+c-brotli = ["shared-brotli-patch-decoder/c-brotli"]
```

Comment:
welcome to one of the annoying corners of rust: features are supposed to be 'additive', that is you can add them in any combination and things work, see https://doc.rust-lang.org/cargo/reference/features.html?highlight=additive#feature-unification.


In any case there's actually a section in the docs now on "mutually incompatible features", so that would be worth a skim at least: https://doc.rust-lang.org/cargo/reference/features.html#mutually-exclusive-features

Reply:
I set up shared-brotli-patch-decoder to work if both are enabled simultaneously (c takes priority) using the cfg-if mechanism as suggested there.